haas/api.py

```diff
@@ -22,6 +22,7 @@
 from schema import Schema, Optional
 
 from haas import model
+from haas.auth import get_auth_backend
```

Comment:
After looking at the code more, I'm starting to think we should establish a convention of using `local` for things that happen all the time, as we did for the DB session handle.

What do you think?